xml/hwbp_registers.xml

```diff
@@ -108,7 +108,13 @@
             corresponds to the maximum NAPOT range, which is $2^{63}$ bytes in
             size.
         </field>
-        <field name="0" bits="XLEN-12:20" access="R" reset="0" />
+        <field name="0" bits="XLEN-12:21" access="R" reset="0" />
+        <field name="hit" bits="20" access="R/W" reset="0">
+            When this trigger matches, this optional bit becomes set. The
+            trigger's user can use it to see which trigger(s) matched, and is
+            responsible for clearing it. If the bit is not implemented, it is
```

Comment:
"is responsible for clearing it" is more of a protocol thing vs how the HW should actually work.

Can it just say:
```
If this optional bit is implemented, the hardware sets it when this trigger matches. The trigger's user can set or clear it at any time.
```

Elsewhere in the text we could explain as a `commentary` thoughts about how its expected to be used by the SW.

Reply:
I'm happy to remove "responsible" as that implies a mandate, but I don't see any benefit in explaining the use somewhere else.